native/src/routes/JpalTracking.tsx

```diff
@@ -88,6 +88,8 @@ const JpalTracking = ({ navigation }: JpalTrackingProps): ReactElement => {
             onPress: () => updateTrackingEnabled(true),
           },
         ])
+      } else {
+        navigation.dispatch(action)
```

Comment:
Otherwise the back navigation is not carried out on back button click/system OS back navigation. Just stumbled upon this while looking for the usages of the `SettingSwitch` component.